src/Util/Autoload.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace BetterReflection\Util;
+
+use BetterReflection\Reflection\ReflectionClass;
+use PhpParser\Node\Name;
+use PhpParser\Node\Stmt\Namespace_;
+use PhpParser\PrettyPrinter\Standard as CodePrinter;
+
+class Autoload
+{
+    /**
+     * @var ReflectionClass[]
+     */
+    private static $reflections = [];
```

Comment:
Make all these instance methods, as per above. Will also serve for easier testing later on :-)